ntbs-integration-tests/xunit.runner.json

```diff
@@ -0,0 +1,3 @@
+{
+    "shadowCopy": false

```

Comment:
Curious what this is? :)

Reply:
This is to ensure that tests run in the same directory as the output directory  (see https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-2.2#disable-shadow-copying) - when it wasn't turned on, debugging didn't work. I'll add a comment linking to the docs!

Reply:
Can't add comments to json whoops

Reply:
Ah, sadly not. But thanks for the link, really useful!